snaplet-i18n.cabal

```diff
@@ -44,6 +44,7 @@ Library
               , snap-loader-static
               , text
               , xmlhtml
+              , map-syntax >= 0.2 && < 0.3
```

Comment:
Well, I don't honestly know how the whole thing works.  I'm not very experienced at Haskell development.  But my understanding is that `stack.yaml` is generated from this `.cabal` file, so I'm not confident that I'm being redundant by having both this line and the line in `stack.yaml`, but I don't mind removing the range, if you don't like it.